src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -51,13 +51,16 @@ library GPv2Encoding {
         uint8 sellTokenIndex;
         uint8 buyTokenIndex;
         uint256 executedAmount;
-        bytes32 digest;
         address owner;
+        bytes orderUid;
     }
 
     /// @dev The stride of an encoded trade.
     uint256 private constant TRADE_STRIDE = 204;
 
+    /// @dev The byte length of an order unique identifier.
+    uint256 private constant ORDER_UID_LENGTH = 56;
```

Comment:
Isn't this also a stride for consistency?

Reply:
To me "stride" is specifically for memory offset between the start of successive array elements. So the stride of `orderUid` would depends how it is represented in memory. Specifically in Solidity it would be `sizeof(length || orderUid) == 32 + 56`. In this case I meant "length" as a byte-length of the array.